### PR TITLE
#SCC-128 Restructure section 2

### PR DESCRIPTION
**Pull request by JohanBester**
Hi Folks, 

I have restructured section 2.1 to place Git at the end of the section.
Placing Git at the end of the class section would allow one to complete all the other work, and then spend all the rest of the time teaching the Git / GitHub work.

Also updated all the section2 activities to place the code in code blocks

Created a  new proposed section 2.2 and merged the work from sections 2.2.1 and 2.2.2 into it.
I did not delete the old sections so that the new combined section may be reviewed first.

Kindly let me know your thoughts on all these changes.

Thanks,
Johan

### CODE REVIEW

File: Section02/2.2-JSdataTypeAndVariables.md [hunk 1]
```diff
@@ -0,0 +1,315 @@
+# JavaScript, Data Types & Variables
+
+### **Objective: -**
+
+In this module we'll learn about ...
+
+- Programing and Computers
```
Comment (matsinet) [inline Section02/2.2-JSdataTypeAndVariables.md:7]: Programming has two "m"s

**Review by matsinet — APPROVED** [review]
Update the one spelling mistake and it is approved.

File: Section02/2.2-JSdataTypeAndVariables.md (hunk 1 above)
Comment (JohanBester) [reply Section02/2.2-JSdataTypeAndVariables.md]: OK I have updated the spelling mistake

**Review by cbrantley4 — APPROVED** [review]
Merge it!

**Review by dadler89 — APPROVED** [review]

**Review by matsinet — APPROVED** [review]
LGTM

**Review by dadler89 — APPROVED** [review]
Looks great Johan!

**Review by cbrantley4 — APPROVED** [review]
Merge it!

**Pull request merged**